### PR TITLE
Print message if user enters a url instead of package name

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
git1

git2

### CODE REVIEW

**Comment by ForgottenProgramme** [comment]
@jaimergp

File: grayskull/pypi/pypi.py [hunk 1]
```diff
@@ -552,6 +552,12 @@ def _get_metadata(self) -> dict:
         version = ""
         if self["package"]["version"].values:
             version = self.get_var_content(self["package"]["version"].values[0])
+        if name.startswith("http://"):
```
Comment (jaimergp) [inline grayskull/pypi/pypi.py:555]: We need to consider https protocols too. A nice but little known feature of `.startswith` is that it supports tuples of strings as an argument to try different things. That way we can write this as:

```suggestion
        if name.startswith(("http://", "https://")):
```

File: grayskull/pypi/pypi.py [hunk 2]
```diff
@@ -552,6 +552,12 @@ def _get_metadata(self) -> dict:
...
+            print_msg(
+                f"Sorry. Grayskull presently only supports PyPI packages."
+            )
+            sys.exit()
```
Comment (jaimergp) [inline grayskull/pypi/pypi.py:559]: Are there other instances of using `sys.exit()` in this codebase? It's usually not a good idea to exit from a library unless it's directly exposed to the CLI (raising an exception + catching it in the CLI layer would work better), but if other parts of the code are doing it too, then I guess it's ok.

**Comment by jaimergp** [comment]
Superseded by #6

**Pull request closed without merging**